vissl/utils/io.py

```diff
@@ -66,7 +66,7 @@ def save_file(data, filename):
         with PathManager.open(filename, "wb") as fopen:
             np.save(fopen, data)
     elif file_ext == ".json":
-        with PathManager.open(filename, "a") as fopen:
+        with PathManager.open(filename, "w") as fopen:
```

Comment:
Thanks so much Pranav for submitting a PR.

The method save_file is used by other parts of VISSL, and as such we need to make sure it works with the other parts. I would suggest the following: 

1. Add option `append`, with default False to #save_file.
2. Add in code to support json append=True and append=False. 
3. Change [here](https://github.com/facebookresearch/vissl/blob/4b19ab28953470d615a25273fa22eab659528867/vissl/hooks/log_hooks.py#L498)) to use append=True. This is the only use of #save_file in the codebase using json format. Global search save_file to validate this.

CC: @prigoyal

Reply:
Continued at #346